sefaria/model/tests/topic_test.py

```diff
@@ -164,6 +165,25 @@ def test_sanitize(self):
         assert "<script>" not in t.description["he"]
         assert "<script>" not in t.slug
 
+    def test_update_topic(self, topic_graph):
```

Comment:
move to `helper/tests/topic_test.py`. create a fixture there that creates a single topic for use in the test